coalib/settings/ConfigurationGathering.py

```diff
@@ -293,6 +301,25 @@ def load_configuration(arg_list,
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    remove_default_section(sections)
+
+    remove_default_section(to_save_sections)
+
+    save_sections(to_save_sections)
+
+    str_log_level = str(sections['cli'].get('log_level', '')).upper()
+    logging.getLogger().setLevel(LOG_LEVEL.str_dict.get(str_log_level,
+                                                        LOG_LEVEL.INFO))
+
+    return sections, targets
+
+
+def remove_default_section(sections):
+    """
+    Removes default section from the sections
+    :param sections:
```

Comment:
Won't it be good to add some little description about `sections` and `return` in the docstring?